### ISSUE TITLE
No more updates planned?

### ISSUE THREAD

**Issue by Eldenroot** [fancyapps/fancybox#2433]
Hi @fancyapps !

I have been using your library for over 3 years... it is great! However no updates since 2019, is it still actively development or what are you plans? Thank you!

One more question - is safe to use these CDN hosted files of FancyBox?
https://cdnjs.com/libraries/fancybox

**Comment by fancyapps** [comment]
Hi,

To be short, here is a preview of what is coming next - http://fancybox.test/next/

About CDN - yes, why not, but I would prefer https://www.jsdelivr.com/package/npm/@fancyapps/fancybox

**Comment by Eldenroot** [comment]
URL is not working :( hovewer I am looking forward!!!

**Comment by fancyapps** [comment]
@Eldenroot Yes. It was a bumpy road, but I am now finally satisfied with the result.

btw, you might think you will be using only Fancybox, but (for image galley) it actually creates two synced Carousel instances (each have Panzoom under the hood) and each slide in the main carousel have their own Panzoom instance (for image panning/zooming) :)

**Comment by rctneil** [comment]
@fancyapps These look really nice!  Is there a preview build we could use?  I'm working on my personal site and am in the need of a really nice, clean, simple, jQuery-free modal and the new fancybox looks like it may well do the trick.

Thanks!

**Comment by Eldenroot** [comment]
@fancyapps - it is really cool, looks nice! Any ETA for first beta?

**Comment by mbuzun** [comment]
@fancyapps great news

**Comment by hexcross** [comment]
@fancyapps big up for coming update!
